examples/java/spring-boot/src/test/java/uk/gov/api/springboot/infrastructure/ScheduledHandlerIntegrationTest.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.api.springboot.infrastructure;
+
+import static org.awaitility.Awaitility.await;
+import static org.mockito.Mockito.atLeast;
+import static org.mockito.Mockito.verify;
+
+import org.awaitility.Durations;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.SpyBean;
+
+@SpringBootTest
```

Comment:
We could alternatively do this - thoughts?

```suggestion
@SpringBootTest(properties="apis-fetch-rate.in.milliseconds = 500")
```

Reply:
yeah that makes it easier to see at a glance what it is for this test